boutdata/data.py

```diff
@@ -955,7 +963,15 @@ def __init__(
         suffix=None,
         caching=False,
         DataFileCaching=True,
-        **kwargs
+        info=False,
+        xguards=True,
+        yguards=False,
+        tind=None,
+        xind=None,
+        yind=None,
+        zind=None,
```

Comment:
Should these be documented as well, or are these part of input to `_caching_collect()`?
If yes, maybe you could write what are the `kwargs` which are not part of `_caching_collect()`?